components/progressbar/stories/progressbar.stories.js

```diff
@@ -1,15 +1,33 @@
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { isIndeterminate, size, staticColor } from "@spectrum-css/preview/types";
 import { ProgressBarGroup } from "./progressbar.test.js";
+import { IndeterminateGroup, Template } from "./template.js";
 
 /**
  * The progress bar component shows the progression of a system operation such as downloading, uploading, processing, etc. in a visual way.
+ * 
+ * ## Values
+ * The value is the progress of a system operation (e.g., downloading, uploading, processing) within the progress bar’s range, from the min value to max value. By default, the min value starts at 0 and the max value is set to 100. These values are not applicable when a progress bar is indeterminate.
+ * 
+ * Progress bars can have a value label at the end of the track that gives detailed information about the progress (e.g. "60%" or "2 of 8"). This value label works alongside the label and should not be displayed if the label itself is not displayed. It should also not be displayed if the progress is indeterminate. Similar to the label, the value label is always placed above the track.
  */
 export default {
 	title: "Progress bar",
 	component: "ProgressBar",
 	argTypes: {
-		customWidth: { table: { disable: true } },
+		customWidth: {
+			name: "Custom width",
+			defaultValue: 192,
+			description: "Adjust the width of the component between 48px and 768px. 192px is the default width on desktop, while 240px is the default on mobile.",
+			type: { name: "number" },
+			table: {
+				type: { summary: "number" },
+				category: "Component",
+				defaultValue: { summary: 192 },
+			},
+			control: { type: "range", min: 48, max: 768,},
```

Comment:
This `customWidth` control for progress bar and meter doesn't reset itself back to the default width if I click on the "Reset controls" button. It's honestly not a huge deal and I'm not seeing anything wrong here (maybe it's an issue with Storybook?), but it was a little annoying and I'm calling it out just on the off-chance that someone knows why this is happening.

Reply:
I missed actually setting the default value in the `args` 🤦‍♀️ https://github.com/adobe/spectrum-css/pull/3186/commits/f383888c6e1b8a2c5f6a93d69301291ee3988d47